src/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -16,5 +15,6 @@
     <Reference Include="netstandard" />
     <Reference Include="System.Memory" />
     <Reference Include="System.Threading.Tasks.Extensions" />
+    <ProjectReference Include="..\..\Microsoft.Bcl.AsyncInterfaces\ref\Microsoft.Bcl.AsyncInterfaces.csproj" />
```

Comment:
nit: Could you add a comment explaining why this is a project reference?

Reply:
all References that are not part of the targetting pack are always Project references on ref/*csprojs. This is to avoid race conditions when building the refs. The ones above are fine as simple `Reference` because they are not built in corefx, they come from a targeting pack instead, but the Bcl.AsyncInterfaces netstandard configuration is built in corefx, hence it needs to be a project reference. As this is the standard used across the repo, I'm not sure if we should add a comment explaining. That said, if you still think it's better, I'll go ahead and add it.